src/main/java/duke/utility/UI.java

```diff
@@ -0,0 +1,39 @@
+package duke.utility;
+
+import duke.exception.DukeException;
+
+import java.util.Scanner;
+
+/**
+ * Handle user input
+ */
+public class UI {
```

Comment:
Is it better to write the class name in PascalCase?
[https://se-education.org/guides/conventions/java/basic.html#naming](url)